Makefile

```diff
@@ -692,7 +692,7 @@ bazel-push-aws-run: bazel-push
 
 .PHONY: bazel-gazelle
 bazel-gazelle:
-	bazel run //:gazelle -- -proto disable
+	hack/update-bazel.sh
```

Comment:
For some reason I was getting different results running bazel native vs vendored.  I figure if CI is using hack, we should use it here too?